### ISSUE TITLE
Hyperparameter setting for Table Retrieval Model

### ISSUE THREAD

**Issue by Doreenruirui** [google-research/tapas#135]
Dear developers,

Could you please share the hyper-parameters used for the table retrieval model to replicate the results [here](https://github.com/google-research/tapas/blob/master/DENSE_TABLE_RETRIEVER.md)? In the paper, you mentioned that you used warmup ratio 0.2 to train the large retrieval model, but in the training script you provided the warmup ratio is 0.01. Also, is the hyper-parameter setting for the medium model the same with the large model? Thanks a lot!

Thanks!

**Comment by SyrineKrichene** [comment]
Hi,
Can you give me the exact name of the model/ models that you want to
replicate, I can look at our experiments and give you the exact hyper
parameters to run.

Thanks,
Syrine
